fetch/api/basic/scheme-data.any.js

```diff
@@ -16,7 +16,7 @@ function checkFetchResponse(url, data, mime, fetchMode, method) {
       assert_equals(resp.headers.get("Content-Type"), mime, "Content-Type is " + resp.headers.get("Content-Type"));
      return resp.text();
     }).then(function(body) {
-        assert_equals(body, data, "Response's body is correct");
+        assert_equals(body, data, method === "HEAD" ? "" : "Response's body is correct");
```

Comment:
I don't think this works as only the first two arguments are compared. I'd suggest changing line 32 to make data the empty string or null or some such.

Reply:
Oops, sorry. Fixed.